apps/data-studio/src/components/app/AppHandler/AppHandler.tsx

```diff
@@ -42,13 +42,16 @@ import {getMe} from '../../../graphQL/queries/userData/me';
 import {initialActiveLibrary, useActiveLibrary} from 'hooks/useActiveLibrary';
 import {ME} from '../../../_gqlTypes/ME';
 import Router from '../../Router';
+import ReactModal from 'react-modal';
 
 function AppHandler(): JSX.Element {
     const {t, i18n} = useTranslation();
     const dispatch = useAppDispatch();
     const {token: themeToken} = theme.useToken();
     const homeUrl = `/${APPS_ENDPOINT}/${APP_ENDPOINT}`;
 
+    ReactModal.setAppElement(document.getElementsByTagName('body')[0]);
```

Comment:
**note:** corrige un warning à l'affichage d'une modal du DS

Reply:
sur AMP il y a un warning au survol du dropdown du header (findDOMNode is deprecated and will be removed in the next major release....), je ne sais pas si c'est le même type de warning que tu as eu Teddy ?

Reply:
Dans le warning qu'on avait, il est dit explicitement qu'il faut rajouter ce `setAppElement` donc c'est autre chose à priori.

Reply:
@P0ppoff, ca ne vient pas d'une mise à jour, la modale du DS se bases sur react-modal, et ce warning a toujours été là